spaces/domains.json

```
@@ -53,6 +53,7 @@
   "vote.bamboodao.io": "bamboodao.eth",
   "vote.moonforcedao.eth": "moonforcedao.eth",
   "vote.pyedao.eth": "pyedao.eth",
+  "vote.pye.net": "pyedao.eth",
```

Could you delete your old subdomain? Thank you
```suggestion
  "vote.pye.net": "pyedao.eth",
```